wnfs-common/src/lib.rs

```diff
@@ -1,10 +1,13 @@
+//! TODO(appcypher): Needs documentation.
```

Comment:
When opening up the docs, this is what I see at the top.

Intended to leave, or do this PR?

Reply:
```suggestion
//! WNFS Utils
//------------------------------------------------------------------------
// Documentation:
//------------------------------------------------------------------------
#![deny(missing_docs)]
#![doc = include_str!("../README.md")]
```